src/styles/blocks/arrows.scss

```diff
@@ -0,0 +1,17 @@
+.arrow {
```

Comment:
It's not critical, but It would be great to name file just like the BEM block: `arrow.scss` instead of `arrowS.scss`